result/neighborsv4.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv4 Neighbors models
```

Comment:
IPv4 Neighbors model.

Reply:
Almost every place, "models" is written (example, bgpv4.yaml, bgpv6.yaml, state.yaml), including otg "models". So, followed the existing descriptions.